core/src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/HttpGoogleServicesDAOSpec.scala

```diff
@@ -461,12 +463,48 @@ class HttpGoogleServicesDAOSpec extends FlatSpec with Matchers with IntegrationT
     }
   }
 
-  it should "fail to get usage when there are no storage logs" in {
-    intercept[GoogleStorageLogException] {
-      Await.result(gcsDAO.getBucketUsage(RawlsBillingProjectName(testProject), "no-bucket-here-" + UUID.randomUUID.toString), Duration.Inf)
+  it should "return 0 usage when there are no storage logs and the bucket is empty" in {
+    val storage = gcsDAO.getStorage(gcsDAO.getBucketServiceAccountCredential)
+
+    val projectName = RawlsBillingProjectName(testProject)
+    val logBucketName = gcsDAO.getStorageLogsBucketName(projectName)
+    val bucketName = "services-dao-spec-" + UUID.randomUUID.toString
+    val bucket = new Bucket().setName(bucketName)
+    val bucketInserter = storage.buckets.insert(testProject, bucket)
```

Comment:
this is not part of our unit tests, it is integration testing which happens for each develop build